test/adapters/flexibleLeverageStrategyAdapter.spec.ts

```diff
@@ -2034,10 +2153,12 @@ describe("FlexibleLeverageStrategyAdapter", () => {
   describe("#ripcord", async () => {
```

Comment:
Can we write a test with 2 exchanges, we call ripcord on both exchanges immediately after one another, where the timestamps are updated correctly